docs/CODE_GUIDELINES.md

```diff
@@ -622,22 +626,47 @@ class Foo
 };
 ```
 
-### 10.1. Destructors in interfaces
+### 11.5. Destructors in interfaces
 
 A class with any virtual functions should have a destructor that is either public and virtual or else protected and nonvirtual (cf. [ISO C++ guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-dtor-virtual)).
 
-## 10. Other conventions
+### 11.6. Constructor Initialzation Lists
+
+For lines up to length `ColumnLimit` everything will be on one line, excluding the braces which must be on the following lines.
```

Comment:
Where do I find info about `ColumnLimit`?

Reply:
In .clang-format. Can add that too.

Reply:
Added